tests/openssl/tests.unsupported

```diff
@@ -0,0 +1,23 @@
+# Trigger abort, always failed.
+../../3rdparty/openssl/openssl/test/aborttest.c
+# Require localtime, which is not supported.
+../../3rdparty/openssl/openssl/test/asn1_time_test.c
+# Require fork, which is not supported.
+../../3rdparty/openssl/openssl/test/drbgtest.c
+# Disable the stress test that takes too long to finish.
```

Comment:
Does this pass at least though, or has it never run to completion?

Reply:
It took around 70 second (tested both on Linux and Windows) to pass so I disable it.